nanoFirmwareFlasher.Library/CC13x26x2Device.cs

```diff
@@ -82,22 +82,22 @@ public ExitCodes FlashHexFiles(IList<string> files)
 
             if (Verbosity == VerbosityLevel.Normal)
             {
-                Console.Write("Flashing device...");
+                OutputWriter.Write("Flashing device...");
             }
             else if (Verbosity >= VerbosityLevel.Detailed)
             {
-                Console.WriteLine("Flashing device...");
+                OutputWriter.WriteLine("Flashing device...");
             }
 
             // program HEX file(s)
             foreach (string hexFile in files)
             {
                 if (Verbosity >= VerbosityLevel.Detailed)
                 {
-                    Console.WriteLine($"{Path.GetFileName(hexFile)}");
+                    OutputWriter.WriteLine($"{Path.GetFileName(hexFile)}");
                 }
 
-                var cliOutput = RunUniflashCli($" flash -c {ConfigurationFile} -f -v {hexFile}");
+                string cliOutput = RunUniflashCli($" flash -c {ConfigurationFile} -f -v {hexFile}");
```

Comment:
Why are all these being changed to explicit declaration instead of `var`?

Reply:
Because the .editorconfig rules suggest that.

Personally I'm more of a "var" guy. So while coding I'll use "var" everywhere. When I'm more or less done, there will be a lot of messages (from the .editorconfig rules) that suggest to change "var" to an explicit type declaration. Not for `var x = new SomeType();`, but always for `var x = func ();`. The editor offers the option to fix these everywhere in the document/project/solution etc. With existing code, I choose for "document". That also affects existing code, but alas, it makes it adhere to your code style, so it should not be too bad.